kafka_enforcer_common/src/main/java/com/tesla/data/enforcer/BaseCommand.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2020. Tesla Motors, Inc. All rights reserved.
+ */
+
+package com.tesla.data.enforcer;
+
+import com.beust.jcommander.IStringConverter;
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.ParameterException;
+import com.beust.jcommander.Parameters;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import tesla.shade.com.google.common.io.Resources;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+/**
+ * All enforcement related CLI tools are an extension of {@link BaseCommand}.
+ */
+@Parameters(commandDescription = "Validate config")
```

Comment:
seems like an odd descrption. does this even need parameters?

Reply:
Yeah, its both a 'base' class and a real command. You can run `validate` and it would just parse the YAML and tell you if it looks good.

```java  
public int run() {
    System.out.println("Kafka connection: " + kafkaConfig().get("bootstrap.servers"));
    System.out.println("Config looks good!");
    return SUCCESS;
  }
```

Reply:
oh neat! +1